src/morale.cpp

```diff
@@ -451,14 +451,16 @@ void player_morale::decay( const time_duration ticks )
     invalidate();
 }
 
-void player_morale::display( int focus_eq )
+void player_morale::display( int focus_eq, int pain_penalty, int fatigue_penalty )
 {
     /*calculates the percent contributions of the morale points,
      * must be done before anything else in this method
      */
     calculate_percentage();
 
     const char *morale_gain_caption = _( "Total morale:" );
```

Comment:
Huh, I wonder why it was not even a `std::string`...

Reply:
It seems to be translated:

```
lang/po/zh_CN.po
171779-#, c-format
171780:msgid "Total morale: %d"
171781-msgstr "总心情值：%d"
```

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/TRANSLATING.md#translation-functions